app_flutter/lib/widgets/task_icon.dart

```diff
@@ -44,22 +44,19 @@ class TaskIcon extends StatelessWidget {
           color: blendFilter,
         );
       case StageName.luci:
-        return Image.asset(
-          'assets/chromium.png',
-        );
-      case StageName.devicelab:
-        return Image.asset(
-          'assets/android.png',
-        );
-      case StageName.devicelabWin:
-        return Image.asset(
-          'assets/windows.png',
-        );
-      case StageName.devicelabIOs:
-        return Image.asset(
-          'assets/apple.png',
-          color: blendFilter,
-        );
+        if (qualifiedTask.builder.toLowerCase().startsWith('linux')) {
```

Comment:
We no longer need the switch for `StageName`, let's remove it

Reply:
done

Reply:
The switch is still here